.travis.yml

```diff
@@ -19,6 +19,11 @@ matrix:
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 7.0
       env: COMPOSER_FLAGS="" SYMFONY_VERSION="3.0.*"
+    - php: 7.1
+      env: COMPOSER_FLAGS="" SYMFONY_VERSION="dev-master"
+  allow_failures:
+    - php: 7.1
```

Comment:
We should verify running 7.1 with the latest stable Symfony works.

It's ok to have the `dev-master` fail, but stables should always work.